src/annotator/util/shadow-root.js

```diff
@@ -43,5 +44,18 @@ export function createShadowRoot(container) {
     applyFocusVisible(shadowRoot);
   }
 
+  stopEventPropagation(shadowRoot);
   return shadowRoot;
 }
+
+/**
+ * Stop bubbling up of the 'click' and 'touchstart' events.
```

Comment:
```suggestion
 * Stop bubbling up of 'click' and 'touchstart' events.
```

Can you add a couple of sentences after the summary about the reason for doing this, for future readers. Can you also add a blank line between the documentation and the param docs. In other words, format JSDoc as:

```js
/**
 * Short summary
 *
 * More detailed notes.
 *
 * @param ...
 */

Reply:
No problem, I will do.